README.md

```diff
@@ -1 +1,84 @@
-# autogluon_bench
\ No newline at end of file
+<div align="left">
+  <img src="https://user-images.githubusercontent.com/16392542/77208906-224aa500-6aba-11ea-96bd-e81806074030.png" width="350">
+</div>
+
+# AutoGluon-Bench
+
+## Setup
+
+```
+git clone https://github.com/suzhoum/autogluon-bench.git
+cd autogluon-bench
+git checkout poc_0.0.1
+
+# create virtual env
+python3.9 -m venv .venv
+source .venv/bin/activate
+
+# install autogluon-bench
+pip install -e .
+```
+
+## Run benchmarkings locally
+
+Currently, `tabular` makes use of the [AMLB](https://github.com/openml/automlbenchmark) benchmarking framework, so we will supply the required and optional arguments when running benchmark for tabular. It can only run on the stable and latest(master) build of the framework at the moment. In order to benchmark on a custom branch, changes need to be made on AMLB.
+
+```
+python ./runbenchmarks.py  --config_file path/to/local_config_file
+```
+A sample local config file is available for reference at `./sample_configs/local_configs.yaml`. All keys should have single value.
+
+Note that `multimodal` benchmarking directly calls the `MultiModalPredictor` without the extra layer of [AMLB](https://github.com/openml/automlbenchmark), so the set of arguments we call is different from that of running `tabular`. Currently, we support benchmarking `multimodal` on custom branch of the main repository or any forked repository.
+
+To customize the benchmarking experiment, including adding more hyperparameters, and evaluate on more metrics, you can refer to `./src/autogluon/bench/frameworks/multimodal/exec.py`.
+
+Results are saved under `$WORKING_DIR/benchmark_runs/$module/{$benchmark_name}_{$timestamp}`
+
+
+## Run benchmarkings on AWS
```

Comment:
What permissions are needed if we want to use aws mode?

Reply:
Currently I assume the `dev-users` role to run in `aws` mode. But any role with permissions to create the stacks, i.e. AWS Batch, S3, Lambda, and ECR should be able to run it.